Resources/Prototypes/CharacterItemGroups/Generic/languageGroups.yml

```
@@ -16,6 +16,18 @@
       id: ValyrianStandard
     - type: trait
       id: Azaziba
+    # Einstein Engines - Tajaran section start
+    - type: trait
+      id: SiikMaas
+    - type: trait
+      id: NalRasan
+    - type: trait
+      id: SiikTajr
+    - type: trait
+      id: YaSsa
+    - type: trait
+      id: Delvahii
+    # Einstein Engines - Tajaran section end
```

```suggestion
    - type: trait
      id: SiikMaas
    - type: trait
      id: NalRasan
    - type: trait
      id: SiikTajr
    - type: trait
      id: YaSsa
    - type: trait
      id: Delvahii
```
We don't actually need these comments in particular since this is our own code.